src/components/CollapsibleArrows/CollapsibleArrows.tsx

```diff
@@ -15,7 +15,7 @@ export const CollapsibleArrows = ({
 }: CollapsibleArrowsPropsType) => {
   return (
     <FontAwesomeIcon
-      className={classNames('ms-auto', className)}
+      className={classNames(className)}
```

Comment:
Redundant `classNames` call since there are no modifiers anymore.

Reply:
kept it this way to avoid having a 'undefined' classname and for easier future customization